device/cpuset_lib.cpp

```diff
@@ -458,7 +458,6 @@ int tt_cpuset_allocator::_get_num_tt_pci_devices_by_pci_device_id(uint16_t devic
     if (m_num_tt_device_by_pci_device_id_map.find(device_id_revision) != m_num_tt_device_by_pci_device_id_map.end()) {
```

Comment:
I don't see any functional changes in this PR, only cosmetic changes. How does this fix #126 , what am I missing?

Reply:
The call to `get_num_tt_pci_devices_by_pci_device_id` in the error path `if (mapping == MAP_FAILED)` is removed.  The `get_num_tt_pci_devices_by_pci_device_id` method was hitting a `log_fatal` because it was passed a bad parameter.